src/app/files/file-explorer.js

```diff
@@ -200,7 +200,8 @@ function fileExplorer (localRegistry, files) {
   }
 
   // register to main app, trigger when the current file in the editor changed
-  self._deps.fileManager.event.register('currentFileChanged', (newFile, explorer) => {
+  self._deps.fileManager.events.on('currentFileChanged', (newFile) => {
```

Comment:
usually we used `event` without `s` . Any specific reason for this change? 
in other case perhaps better to keep `event` (even if the `event` interface has changed)

Reply:
I switch to `events` because this is the interface expected by the AppManager to listen and broadcast to the other modules and plugins.
`events` is also exposed in the `Plugin` class and matchs the `Profile` interface.
Also it would help making the difference between `event` => `EventManager` and `events` => `EventEmitter`. Might be useful for the migration.

Reply:
ok